Public/Assets/Js/reboot.js

```diff
@@ -299,10 +299,63 @@ var planningController = function (idElement, options, creneaux)
     this.typeSemaine = this.options['typeSemaine'];
     this.debut = this.options['typeHeureDebut'];
     this.fin   = this.options['typeHeureFin'];
+    this.creneauxList = [];
     this.incrementMatin = 0;
     this.incrementApresMidi = 0;
 
     /**
+     * Fonction appelé à chaque modification (ajout/suppression) de période du planning
+     * @return void
+     */
+    this._changePeriod = function () {
+        // On calcul et affiche la durée de travail hebdomadaire
+        var minuteHedbo = this._calculDureeHebdomadaire();
+        this._affichageDureeHebdomadaire(minuteHedbo);
+    }
+    
+    /**
+     * Affiche la durée de travail hebdomadaire, utilise l'option dureeHebdoId passée lors de la création de l'objet planningController
+     * @param  {int} minuteHedbo Le nombre de minutes travaillées par mois
+     * @return void
+     */
+    this._affichageDureeHebdomadaire = function (minuteHedbo) {
+        if(typeof this.options['dureeHebdoId'] === "undefined") {
+            return;
+        }
+        var diffHrs = Math.floor(minuteHedbo / 60); // heures
+        var diffMins = Math.round(minuteHedbo % 60); // minutes
+        diffMins == 0 ? diffMins = '' : diffMins = diffMins + 'm';
+        document.getElementById(this.options['dureeHebdoId']).innerHTML = diffHrs + 'h ' + diffMins;
+    }
+
+    /**
+     * Calcul le temps de travail hebdomadaire du planning
+     * Utilise le model creneauxList
+     * @return {int} minuteHedboLe nombre de minutes travaillées par mois
+     */
+    this._calculDureeHebdomadaire = function () {
+        var minuteHedbo = 0;
+        var creneau, heureFin, heureDebut;
+        var dFin = new Date();
+        var dDebut = new Date();
+        for (var i in this.creneauxList) {
```

Comment:
Je sais bien que les variables itératrices `i` et `j` sont des conventions, mais le manque de sémantique me gêne.